src/data_provider/src/sysInfoWin.cpp

```diff
@@ -779,12 +810,27 @@ void SysInfo::getProcessesInfo(std::function<void(nlohmann::json&)> callback) co
 
 void SysInfo::getPackages(std::function<void(nlohmann::json&)> callback) const
 {
-    getPackagesFromReg(HKEY_LOCAL_MACHINE, UNINSTALL_REGISTRY, callback, KEY_WOW64_64KEY);
-    getPackagesFromReg(HKEY_LOCAL_MACHINE, UNINSTALL_REGISTRY, callback, KEY_WOW64_32KEY);
+    std::set<nlohmann::json> packages;
+
+    auto fillList{ [&packages](nlohmann::json & data)
```

Comment:
Rename it again as `callback` to avoid unnecessary authorship changes to related functions